dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+| `body`        | Required | Object                                | Text of the ad copy.  For properties, see [`native.body`](#native-body).                                                          |
+| `sponsoredBy` | Required | String                                | The name of the brand associated with the ad.  For properties, see [`native.sponsoredBy`](#native-sponsoredby).                   |
+| `icon`        | Optional | Object                                | The brand icon that will appear with the ad.  For properties, see [`native.icon`](#native-icon).                                  |
+| `image`       | Optional | Object                                | A picture that is associated with the brand, or grabs the user's attention.  For properties, see [`native.image`](#native-image). |
+| `clickUrl`    | Optional | Object                                | Where the user will end up if they click the ad.  For properties, see [`native.clickUrl`](#native-clickurl).                      |
+| `cta`         | Optional | Object                                | *Call to Action* text, e.g., "Click here for more information".  For properties, see [`native.cta`](#native-cta).                 |
+
+<div class="alert alert-warning">
+  <strong>Prebid Native Ad Validation</strong>
+  <p>
+   Prebid.js validates the assets on native bid responses like so:
+  <ul>
+      <li>
+       If the asset is marked as "required", it checks the bid response to ensure the asset is part of the response
+      </li>
+      <li>
+       However, Prebid.js does not do any additional checking of a required asset beyond ensuring that it's included in the response; for example, it doesn't validate that the asset has a certain length or file size, just that that key exists in the response JSON
+      </li>
+    </ul>
+  </p>
+</div>
+
+<a name="native-image" />
+
+##### `adUnit.mediaTypes.native.image`
+
+{: .table .table-bordered .table-striped }
+| Name            | Scope    | Type                                  | Description                                                                                                                         |
+|-----------------+----------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------|
+| `required`      | Required | Boolean                               | Whether this asset is required.                                                                                                     |
+| `sizes`         | Optional | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                             |
+| `aspect_ratios` | Optional | Array[Object]                         | Instead of `sizes`, you can define allowed aspect ratios.  For properties, see [`image.aspect_ratios`](#native-image-aspectratios). |
```

Comment:
You say "instead", but can "aspect_ratios" be defined in addition to "sizes"?

Reply:
Looks like the answer is yes: https://github.com/prebid/Prebid.js/blob/master/test/spec/unit/core/adapterManager_spec.js#L836